lib/jira/resource/agile.rb

```diff
@@ -31,6 +31,18 @@ def self.get_sprint_issues(client, sprint_id, options = {})
         parse_json(response.body)
       end
 
+      def self.get_projects_full(client, board_id, options = {})
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project/full")
+        parse_json(response.body)
+      end
+
+      def self.get_projects(client, board_id, options = {})
+        options[:maxResults] ||= 100
+        options[:startAt] ||= 0
+        response = client.get("/rest/agile/1.0/board/#{board_id}/project?maxResults=#{options[:maxResults]}&startAt=#{options[:startAt]}")
```

Comment:
You should use `path_base(client)` instead of `/rest/agile/1.0` (as used in the other methods)